communication/moderation.md

```diff
@@ -0,0 +1,53 @@
+# Moderation on Kubernetes Communications Channels
+
+This page describes the rules and best practices for people chosen to moderate Kubernetes communications channels. 
+This includes, Slack and the mailing lists and _any communication tool_ used in an official manner by the project. 
+
+## Roles and Responsibilities
+
+As part of volunteering to become a moderator you are now representative of the Kubernetes community and it is your responsibility to remain aware of your contributions in this space. 
+These responsibilities apply to all Kubernetes official channels. 
+
+Moderators _MUST_:   
+
+- Take action as specified by these Kubernetes Moderator Guidelines.
+- Abide by the documented tasks and actions required of moderators.
+- Ensure that the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) is in effect on all official Kubernetes communication channels.
+- Become familiar with the [Kubernetes Community Values](https://github.com/kubernetes/steering/blob/master/values.md).
+- Take care of spam as soon as possible, which may mean taking action by removing a member from that resource.
+- Foster a safe and productive environment by being aware of potential multiple cultural differences between Kubernetes community members.
+- Understand that you might be contacted by moderators, community managers, and other users via private email or a direct message. 
+- Report egregious behavior to steering@k8s.io.
```

Comment:
Isn't there a private list?

Reply:
@jdumars nope, there's a steering-private for the sensitive cases.